lib/action_cable/server/base.rb

```diff
@@ -36,11 +36,11 @@ def worker_pool
         @worker_pool ||= ActionCable::Server::Worker.pool(size: config.worker_pool_size)
       end
 
-      # Requires and returns an array of all the channel class constants in this application.
-      def channel_classes
+      # Requires and returns an hash of all the channel class constants in this application.
+      def channel_classes_hash
```

Comment:
Agreed.

I'd suggest some doc changes while we're here:

```ruby
# Requires and returns a hash of all channel class constants keyed by channel name.
```

(I'd leave out `in this application` as `channel_classes` returning classes for another application would be strange to me.)